test/oauth-authorization-test.js

```diff
@@ -335,24 +351,38 @@ suite.addBatch({
                         requestToken(cl, this);
                     },
                     function(err, rt) {
+                        var self = this;
+
                         if (err) throw err;
-                        br = new Browser({runScripts: false});
-                        br.visit("http://localhost:4815/oauth/authorize?oauth_token=" + rt.token, this);
+                        br = new Browser({runScripts: false}); //faster without scripts...
+                        Step(
+                            function(){
+                        	    br.visit("http://localhost:4815/oauth/authorize?oauth_token=" + rt.token, this);
+                            },
+                            function(){
+                                self(!br.success,br);
+                            }
+                        );
                     },
                     function(err, br) {
+                        var self = this;
+
                         if (err) throw err;
-                        if (!br.success) throw new Error("Browser error");
-                        br.fill("username", "dormouse", this);
-                    },
-                    function(err) {
-                        if (err) throw err;
-                        br.fill("password", "BADPASSWORD", this);
-                    },
-                    function(err) {
-                        if (err) throw err;
-                        br.pressButton("#authenticate", this);
+                        br.assert.success();
+
+                        br
+                            .fill("username", "dormouse", this)
```

Comment:
so
```
br.fill(a);
br.fill(b);
br.fill(c);
```
infavour of
```
br
    .fill(a)
    .fill(b)
    .fill(c)
```

The object returns itself so you can chain function calls.